.github/workflows/ci.yml

```diff
@@ -31,10 +37,12 @@ jobs:
           echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
           sudo apt update
           sudo apt remove kurtosis-cli
-          sudo apt install kurtosis-cli=1.8.1 -V
+          sudo apt install kurtosis-cli=1.11.1 -V
       - name: Download optimism-preimage-maker
         run: |
           git clone https://github.com/datachainlab/optimism-preimage-maker
+          cd optimism-preimage-maker
+          git checkout 28e3b40071881112c739620d298585535a253318
```

Comment:
This commit hash will be replaced with the version tag that will be created after [this PR](https://github.com/datachainlab/optimism-preimage-maker/pull/23) is merged, right?